Sources/Sentry/SentrySpan.m

```diff
@@ -110,6 +110,13 @@ - (instancetype)initWithContext:(SentrySpanContext *)context
         }
 #endif // SENTRY_TARGET_PROFILING_SUPPORTED
     }
+
+    if (_parentSpanId == context.parentSpanId) {
```

Comment:
`m`: Xcode Analyzer seems to leave a warning here:

> Access to instance variable '_parentSpanId' results in a dereference of a null pointer (loaded from variable 'self') [core.NullDereference]

Reply:
Oof, i went to fix the analyzer warning and changed the wrong thing. this should read `if (context.parentSpanId == nil)`, fixed